src/bos/operators/power_operator_base.py

```diff
@@ -103,6 +103,8 @@ def _power_components(self, components: List[dict]) -> List[dict]:
                     LOGGER.debug(f"Acting on component {component}")
                     errors = self._my_power([component])
                     if errors.error_code != 0:
+                        LOGGER.debug("Component %s error code=%s message=%s", component,
```

Comment:
You're logging what should be available via the components endpoint.

Reply:
Yes. This debugging is motivated by the assumption that you have a log file and did not collect that data at the time. In particular, during one of these big BOS sessions, each component goes through a bunch of state changes. It's not feasible to track those just by continually pinging the endpoint, especially since there's no guarantee you wouldn't miss some. That's part of why I'm adding some of this stuff.

Reply:
It's a DEBUG statement; go nuts with these.